lib/dialects/mysql/query-generator.js

```diff
@@ -265,6 +266,16 @@ var QueryGenerator = {
 
     if (attribute.references) {
       attribute = Utils.formatReferences(attribute);
+
+      if (options && options.context === 'addColumn' && options.foreignKey) {
+        var attrName = options.foreignKey;
+
+        template += Utils._.template(', ADD CONSTRAINT <%= fkName %> FOREIGN KEY (<%= attrName %>)')({
+          fkName: this.quoteIdentifier(attrName + '_foreign_idx'),
```

Comment:
I follow the ```changeColumnQuery: function(tableName, attributes) ``` with reference, at
lib/dialects/mysql/query-generator.js : Line 131

``` javascript
constraintString.push(Utils._.template('<%= fkName %> FOREIGN KEY (<%= attrName %>) <%= definition %>')({
          fkName: this.quoteIdentifier(attributeName + '_foreign_idx'),
          attrName: this.quoteIdentifier(attributeName),
          definition: definition.replace(/.+?(?=REFERENCES)/,'')
        }));
```